installation/language/en-GB/joomla.ini

```
@@ -114,6 +114,10 @@ INSTL_SITE="Main Configuration"
 INSTL_SITE_DEVMODE_LABEL="We detected development mode"
 INSTL_SITE_NAME_DESC="Enter the name of your Joomla site."
 
+; Expert View
+INSTL_PUBLIC_FOLDER_LABEL="Relative or absolute path to the public folder"
+INSTL_PUBLIC_FOLDER_DESC_SHORT="Relative or absolute path to the public folder"
```

```suggestion
INSTL_PUBLIC_FOLDER_DESC_SHORT="Relative or absolute path to the public folder"
INSTL_PUBLIC_FOLDER_LABEL="Relative or absolute path to the public folder"
```

alpha sort - but do we really need this description?

---

I would like to have a better description what the user has to do with the webserver so it works, but didn't was able to write something useful yet.

---

ok